pkgs/os-specific/linux/wooting-udev-rules/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "wooting-udev-rules";
+  version = "20190601";
+
+  src = [ ./wooting.rules ];
```

Comment:
It would be nice to have an external url for this that would be updated when for instance a new model is released.

Reply:
Added a comment with the source of the rules.